client/src/modules/debtors/groups.update.html

```diff
@@ -19,6 +19,17 @@
             </div>
           </div>
 
+          <div class="form-group" ng-class="{'has-error' : debtorGroup.locked.$invalid && debtorGroup.$submitted}">
```

Comment:
There is no need to put this in a form group - there isn't any way this can be an error.  Go ahead and remove the `form-group` and the `help-block`.